github/pulls_comments.go

```diff
@@ -156,6 +156,30 @@ func (s *PullRequestsService) CreateCommentInReplyTo(ctx context.Context, owner
 	return c, resp, nil
 }
 
+// CreateReviewCommentReply creates a reply to a review comment for a pull request.
+//
+// GitHub API docs: https://developer.github.com/v3/pulls/comments/#create-a-review-comment-reply
+func (s *PullRequestsService) CreateReviewCommentReply(ctx context.Context, owner string, repo string, prNumber int, body string, commentID int64) (*PullRequestComment, *Response, error) {
```

Comment:
s/owner string, repo string/owner, repo string/

Reply:
Sorry, I didn't get this.